packages/react-slider/src/components/Slider/useSliderState.tsx

```diff
@@ -344,15 +357,26 @@ export const useSliderState = (state: SliderState) => {
     const marksValue = markValues;
     const marksChildren: JSX.Element[] = [];
     for (let i = 0; i < marksPercent.length; i++) {
+      const marksItem = typeof marks === 'boolean' ? null : marks![i];
```

Comment:
You could do the following so that you don't have to use `!` in the code:

```suggestion
      const marksItem = typeof marks === 'boolean' || marks === undefined ? null : marks[i];
```